### PR TITLE
Sparkml

### PR DESCRIPTION
**Pull request by jeffsaremi**
This is the initial version and experimental effort in converting Spark ML models to ONNX
Currently only 4 operators are supported:
- StringIndexer
- OneHotEncoderEstimator
- LogisticRegression
- VectorAssembler

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,11 +5,12 @@ tensorflow
...
 pandas
-pytest
 pytest-cov
 scikit-learn
 scipy
 svm
 wheel
 xgboost
 lightgbm
+pyspark==2.3.2
```
Comment (wenbingl) [inline requirements-dev.txt]: any reason we have to stick to this version?

File: onnxmltools/convert/sparkml/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .convert import convert
+from .utils import *
```
Comment (wenbingl) [inline onnxmltools/convert/sparkml/__init__.py]: can we avoid * import here? pylint always suggest we should not use import *

Comment (jeffsaremi) [reply onnxmltools/convert/sparkml/__init__.py]: I'll change it and make it specific

File: requirements-dev.txt (hunk 1 above)
Comment (jeffsaremi) [reply requirements-dev.txt]: version 2.4 does not work. Until they provide fixes for that yes we need to stick with this

**Pull request merged**